test/suite/test_oligarch09.py

```diff
@@ -60,7 +60,7 @@ class test_oligarch09(wttest.WiredTigerTestCase, DisaggConfigMixin):
     nitems = 1000
 
     def session_create_config(self):
-        cfg = 'key_format=S,value_format=S,block_compressor={}'.format(self.block_compress)
+        cfg = 'delta_pct=200,key_format=S,value_format=S,block_compressor={}'.format(self.block_compress)
```

Comment:
Maybe add a comment explaining the value 200, as I'm assuming you'd like to allow deltas larger than a page here? Without it, "200%" looks like a mistake.

Reply:
You're right, added a comment in 90005a3.